src/rest/endpoints.ts

```diff
@@ -340,8 +341,27 @@ export class RESTEndpoints {
   /**
    * Returns an [audit log](#DOCS_RESOURCES_AUDIT_LOG/audit-log-object) object for the guild. Requires the 'VIEW_AUDIT_LOG' permission.
    */
-  async getGuildAuditLog(guildId: string): Promise<AuditLogPayload> {
-    return this.rest.get(`/guilds/${guildId}/audit-logs`)
+  async getGuildAuditLog(
+    guildId: string,
+    params: {
+      userId?: string
+      actionType?: AuditLogEvents
+      before?: string
+      limit?: number
+    }
+  ): Promise<AuditLogPayload> {
+    let q = ''

```

Comment:
I think it would be better to use searchParams than q

Reply:
I think query(or q) makes more sence